.github/workflows/ci.yaml

```diff
@@ -404,8 +404,8 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{ steps.create_gh_release.outputs.upload_url }}
-          asset_path: ./dist/target/dotty-${{ env.RELEASE_TAG }}.tar.gz
-          asset_name: dotty-${{ env.RELEASE_TAG }}.tar.gz
+          asset_path: ./dist/target/scala3-${{ env.RELEASE_TAG }}.tar.gz
+          asset_name: scala-${{ env.RELEASE_TAG }}.tar.gz
```

Comment:
Does this mean that the name of the tarball referenced in the uploaded `sha256sum.txt` file will be different from the name of the file posted for download?  i.e. the checksum file will say `scala3-3.0.0-M2.tar.gz` but the file will actually be `scala-3.0.0-M2.tar.gz`?

Reply:
Yes, exactly. This is a good reason to align names.

Reply:
Thanks for pointing out!